report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html

```diff
@@ -30,24 +30,42 @@
     <div class="form-group sidepanel-search-section">
       <input type="text" placeholder="Search..." data-aet-sidepanel-search
              data-ng-model="$root.searchText" class="search-input">
-      <i class="glyphicon glyphicon-search" data-toggle="popover"
-         data-content="Start typing to filter by url name" data-trigger="hover"
-         data-container="body"></i>
-      <i class="glyphicon glyphicon-remove" data-toggle="popover" data-content="Clear search"
-         data-trigger="hover" data-container="body" ng-click="clearSearch()"
-         style="display: none;"></i>
+      <div class="fontawesome"
```

Comment:
Are we fine with making that one popover instead of two? And again - do we need those wrappers?